### PR TITLE
Vignesh patch

### PR DESCRIPTION
**Pull request by neshvig10**

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @neshvig10!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: CHANGES.md [hunk 1]
```diff
@@ -4510,3 +4510,5 @@ _This releases fixes 2D and other issues with Chrome 29.0.1547.57 ([#1002](https
 ### a0 - 2011-06-17
 
 - Initial Release.
+- 
+-fixed a typo in line 88 of PolylineCommon.glsl
```
Comment (jjhembd) [inline CHANGES.md:4514]: This change should be listed at the top of the file, under "Fixes" in the 1.108 release. 
Also, I don't think this is really a typo. The old code was correct, but your code will be faster. Perhaps you could say "optimized angle calculations in Polyline shader"

File: Source/Shaders/PolylineCommon.glsl [hunk 2]
```diff
@@ -85,7 +85,7 @@ vec4 getPolylineWindowCoordinatesEC(vec4 positionEC, vec4 prevEC, vec4 nextEC, f
     else {
         lineDir = normalize(nextWindow.xy - positionWindow.xy);
     }
-    angle = atan(lineDir.x, lineDir.y) - 1.570796327; // precomputed atan(1,0)
+    angle = czm_fastApproximateAtan(lineDir.x, lineDir.y) - 1.570796327; // precomputed czm_fastApproximateAtan(1,0)
```
Comment (jjhembd) [inline Source/Shaders/PolylineCommon.glsl:88]: If we replace the constant `1.570796327` with `czm_piOverTwo`, the code will be more self-explanatory, so the comment won't be necessary.

**Review by jjhembd — CHANGES_REQUESTED** [review]
Thanks @neshvig10, this should be a good optimization! I just have 2 suggestions.

**Pull request closed without merging**